### ISSUE TITLE
Stored app data issue

### ISSUE THREAD

**Issue by VasilisKos** [Lucchetto/SuperImage#39]
I've noticed that, everytime an image is opened in the app, the app's size is increased by the data size of the image. Thus, the app's size is increased more and more by time! The bad thing is that these data are stored in the app's data, not its cache. So:
1. Is it necessary that these data are stored? I think after an upscale is over or cancelled, the data should be deleted after exiting the app.
2. If the data are necessary, they should be stored in the app's cache.

**Comment by Lucchetto** [comment]
Hi, the temp image file is cleared when you load another image, can you confirm this ?

**Comment by VasilisKos** [comment]
No, it's not cleared. Follow this actions to reproduce: open an image > exit the app > check app's size in Android settings > open the app and another image > exit image and check again size. Every time it's added on top of the previous.
Maybe it has to do with the fact that it's stored in the app's data, not the cache. That's the one that's increased. This should be changed.

**Comment by Lucchetto** [comment]
The app attempts to delete the temp images directory on startup https://github.com/Lucchetto/SuperImage/blob/master/app/src/main/java/com/zhenxiang/superimage/work/RealESRGANWorkerManager.kt#L35

Which ROM are you running ? Root ?

**Comment by VasilisKos** [comment]
Stock Android 11, on a Nokia 6.2. No root.

**Comment by VasilisKos** [comment]
> The app attempts to delete the temp images directory on startup https://github.com/Lucchetto/SuperImage/blob/master/app/src/main/java/com/zhenxiang/superimage/work/RealESRGANWorkerManager.kt#L35
> 
> Which ROM are you running ? Root ?

Yeah, you're right. Exiting the app through the back button and relaunching it doesn't delete the images. That's what I did. Killing the app through the task manager though, and relaunching it works as you say. Thanks! 👍🏻
